api/clients/v2/verification/cert_verifier.go

```diff
@@ -189,21 +143,16 @@ func (cv *CertVerifier) VerifyCertV2(
 // behind.
 func (cv *CertVerifier) GetNonSignerStakesAndSignature(
 	ctx context.Context,
-	// the hex address of the EigenDACertVerifier contract
-	certVerifierAddress string,
 	signedBatch *disperser.SignedBatch,
 ) (*verifierBindings.NonSignerStakesAndSignature, error) {
-	signedBatchBinding, err := SignedBatchProtoToBinding(signedBatch)
+	signedBatchBinding, err := coretypes.SignedBatchProtoToBinding(signedBatch)
 	if err != nil {
 		return nil, fmt.Errorf("convert signed batch: %w", err)
 	}
 
-	err = cv.MaybeWaitForBlockNumber(ctx, signedBatch.GetHeader().GetReferenceBlockNumber())
-	if err != nil {
-		return nil, fmt.Errorf("wait for block number: %w", err)
-	}
+	blockNumber := signedBatch.GetHeader().GetReferenceBlockNumber()
```

Comment:
```suggestion
	referenceBlockNumber := signedBatch.GetHeader().GetReferenceBlockNumber()
```

Reply:
RBN as a concept is very meaningful to most eigenda engineers, so good to use it as the variable name